pkg/config/env.go

```diff
@@ -18,7 +18,7 @@ var ENV = struct {
 	DBDriver                  string        `env:"OPV_DB_DRIVER" envDefault:"sqlite3"`
 	DBSetupRetryAttempts      uint          `env:"OPV_DB_SETUP_RETRY_ATTEMPTS" envDefault:"9"`
 	DBSetupRetryDelay         time.Duration `env:"OPV_DB_SETUP_RETRY_DELAY" envDefault:"100ms"`
-	DBConnectionStr           string        `env:"OPV_DB_CONNECTION_STR" envDefault:"_opv.sqlite?cache=shared&_fk=1"`
+	DBConnectionStr           string        `env:"OPV_DB_CONNECTION_STR" envDefault:"file:memdb1?mode=memory&cache=shared&_fk=1"`
```

Comment:
made a change to the default in-memory sqlite db, better for testing, and `make run` will have its own OPV_DB_CONNECTION_STR pointing to a local file.